EIPS/eip-3333.md

```diff
@@ -0,0 +1,51 @@
+---
+eip: 3333
+title: Increase block rewards to 3 ETH, with 2 Year Decay to 1 ETH Scheduled
+author: Michael D. Carter (BitsBeTrippin)
+discussions-to: https://discord.gg/3sjBbWKM4E (ETH R&D 1559-mining)
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-03-11
+---
+
+## Simple Summary
+Changes the block reward paid to proof-of-work (POW) miners to 3 ETH from existing 2 ETH, but kicks off a 0.25 quarterly drop to a final 1.0 ETH Reward 2 Years Later.
+
+## Abstract
+Starting with `FORK_BLKNUM` block rewards will be increased to a base of 3 ETH, uncle and nephew rewards will be adjusted accordingly. Every 3 months after, measured by EPOCH count, Block Reward drops 0.25 every quarter till Ethereum Block Reward reaches 1.0 BR per Block.
+
+## Motivation
+With the passing of EIP-1559, the potential steep reduction of Ethereum Miner fee's, which currently are averaging a doubling of the exiting block reward have several scenerios where it can become the 50% key factor on keeping ethereum mining participation growing the current all time high hashpower, all the way to the London hard fork release. There are several scenerios where this can create a steep hashrate decrease when the BASEFEE starts to get burned vs distributed to miners. This can create an exteciential risk to ethereum's network by incentivizing miners to move to more profitable solutions, including broker services such as Nicehash and Miningrigrentals. To ward off this threat, a recommend action is to temporarly increase Eth Mining Rewards by +1 offset then began to decrease -2 Eth over 8 quarters, 2 full years from London's Hardfork. This should align well with the current optimisitic ETH 2.0 PoW/PoS handoff and appropriately set Eth Inflation on a managed reduction against a schedule that can be planned for. Additionally this provides a higher incentive for price discovery which can offset any further steep impacts for the mining security budget. 
```

Comment:
Try to avoid mentioning London or other things related to mainnet or current state of Ethereum in EIPs.  Consider that an EIP may be implemented at any time in the future.  Perhaps reword this to say something along the lines of:

> A sudden drop in PoW mining rewards could result in a sudden precipitous decrease in mining profitability that may drive miners to auction off their hashrate to the highest bidder while they figure out what to do with their now "worthless" hardware.  If enough hashrate is auctioned off in this way at the same time, an attacker will be able to rent a large amount of hashing power for a short period of time at relatively low cost and potentially attack Ethereum.  By setting the block reward to X (where X is enough to offset the sudden profitability decrease) and then decreasing it over time to Y (where Y is a number below the sudden profitability decrease), we both avoid introducing long term inflation while at the same time spreading out the rate that individual miners cross into the unprofitable range.  This allows miners time to sell off/repurpose their hardware rather than a large amount of hashing power being put up for auction at the same time while people try to figure out what to do with their now "worthless" hardware.

Reply:
Good add